doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+.. _ncs_release_notes_150:
+
+|NCS| v1.5.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Bluetooth mesh is supported for development in |release| for prototyping and evaluation.
+Zigbee support for nRF53 Series devices is available for development in |release| for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+Highlights
+**********
+
```

Comment:
Introduced support for `Edge Impulse`_ machine learning models

Reply:
Shall I add this to the highlights, @jdstapes?

Reply:
Ok.

Reply:
Added